crates/bevy_core_pipeline/src/skybox/mod.rs

```diff
@@ -34,7 +38,10 @@ impl Plugin for SkyboxPlugin {
     fn build(&self, app: &mut App) {
         load_internal_asset!(app, SKYBOX_SHADER_HANDLE, "skybox.wgsl", Shader::from_wgsl);
 
-        app.add_plugins(ExtractComponentPlugin::<Skybox>::default());
+        app.add_plugins((
+            ExtractComponentPlugin::<Skybox>::default(),
+            UniformComponentPlugin::<SkyboxUniforms>::default(),
+        ));
```

Comment:
@JMS55 and these… please go through this PR and make another PR against my branch to remove everything that isn’t related to exposure settings.